pkg/cortex/serve/cortex_internal/lib/api/api.py

```diff
@@ -50,7 +51,11 @@ def __init__(
 
         host_ip = os.environ["HOST_IP"]
         datadog.initialize(statsd_host=host_ip, statsd_port=9125)
-        self.statsd = datadog.statsd
+        self.__statsd = datadog.statsd
```

Comment:
nit: IIRC, single underscore is for indicating private variables. Is there a convention for double underscore?

Reply:
double underscore throws an error if you try to use it